README.md

```diff
@@ -105,3 +105,53 @@ If you should want to run from the command line, it is `java -ea -cp build-lib/s
    - Test nginx is routing your local java SoGiveServer: http://local.sogive.org/manifest
    - Test your local web-app: http://local.sogive.org/
 
+## Running the tests
+
+### Puppeteer tests
+
+Jest Puppeteer tests live in `src/puppeteer-tests/__tests__` and exercise core functionality
+of the website, by clicking on elements and checking that other elements appear as expected.
+
+Most of the tests require account details in order to log in to SoGive. To supply these, create a
+file called `puppeteer.credentials.js` in `../logins/sogive-app` (relative to this repository
+root) containing valid account details. For example (dummy values):
+
+	const username = 'foo@example.com';
+	const password = 'password1';
+
+	module.exports = {
+		username,
+		password,
+	};
+
+*Note*, most tests require an account with edit rights.
+
+To run tests, after running `npm install` once to install test dependencies, from the root of the
+repository, run:
+
+	__CONFIGURATION='{"site": "[local/test/prod]"}' npm test [--test file-name.js] [-- -t 'Test case name'] 
+
```

Comment:
Hm -- There's a script runtest.js which does much the same:

`node runtest.js` runs the tests

`node runtest.js --support` gives help on options

`__CONFIGURATION='{"site": "local"}' npm test` has the advantage of colour-styling the output. Otherwise the behaviour is identical.

We should standardise on one. I lean towards `node runtest.js` as it has built-in `--support` info.

FYI: I didn't code runtest.js

Reply:
I discovered that adding `process.env.FORCE_COLOR = true` to `runtest.js` fixes the color issue \o/ 

Will update the PR with that and standardise on runtest.js too

thanks Daniel!

Reply:
just noticed I also need to fixup how runtest.js interprets booleans (right now setting --head false is interpreted as true). I think the whole yargs parsing could be simplified (in fact i implemented this in a previous commit before deciding i was overcomplicating things)

which i'd noticed runtest.js before starting this endeavour! oh well

Reply:
re > right now setting --head false is interpreted as true

Actually, looks like the intention of those flags in the script is that they are true if present and false if absent, so nevermind

Reply:
Done